verification/curator-service/api/test/model/data/source.full.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "source name",
+    "origin": {
+        "url": "foo.bar",
+        "license": "MIT"
+    },
+    "format": "PDF",
+    "automation": {
```

Comment:
missing regexParsing?

Reply:
The reason it isn't here is that regexParsing should be mutually exclusive with a dedicated parser Lambda. Sorry, this is underexplained in this change; another one for the validation work.